packages/flutter/lib/src/cupertino/tab_view.dart

```diff
@@ -125,6 +126,12 @@ class CupertinoTabView extends StatefulWidget {
   /// This list of observers is not shared with ancestor or descendant [Navigator]s.
   final List<NavigatorObserver> navigatorObservers;
 
+  /// Restoration ID to save and restore the state of the [Navigator] built by
+  /// this [CupertinoTabView].
+  ///
+  /// {@macro flutter.widgets.navigator.restorationScopeId}
```

Comment:
How does one unfamiliar with State Restoration go about learning about how to use it? I'm thinking that naturally, developers look for code samples to play around with how to use this. I personally used the test code in this PR to familiarize myself with how to use state restoration with the Navigator/CupertinoTabView.

There is a code sample at [Navigator.restorablePush](https://master-api.flutter.dev/flutter/widgets/Navigator/restorablePush.html) and [Navigatoor.restorablePushNamed](https://master-api.flutter.dev/flutter/widgets/Navigator/restorablePushNamed.html), so maybe we could link there?

Reply:
Good idea. I added three more links to the "see also" section in the macro.